guides/hack/03-expressions-and-operators/99-XHP-attribute-selection.md

```diff
@@ -0,0 +1,17 @@
+When working with [XHP](/hack/XHP/introduction), use the `->:` operator to retrieve an XHP Class [attribute](/hack/XHP/basic-usage#attributes) value.
+
+You can also use `$this->:` in place of the named class instance.
```

Comment:
Why does `$this` get called out here? `->:` can work on arbitrary expressions that resolve to an xhp object. E.g. function calls and operations like ($a ?? $b)->:.

Reply:
Just providing additional context in case a given reader doesn't know of the alternative option. Will add your note and remove the special mention.